src/components/molecules/AppReferralForm.tsx

```diff
@@ -1,14 +1,15 @@
-import useUploadReferral from "../../hooks/useUploadReferral";
-import {useTranslation} from "react-i18next";
-import {namespaces} from "../../i18n/i18n.constants";
-import AppNextButton from "../atom/AppNextButton";
-import ReferralFormProps from "../../types/ReferralFormProps";
-import ReferralFileUploaded from "./ReferralFileUploaded";
-import ReferralNoFileUploaded from "./ReferralNoFileUploaded";
-import { useNavigate } from "react-router-dom";
-import { AppRoute } from "../../enums/app-route";
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
what does this do here?

Reply:
This was for eslint to ignore the unused 'link' variable, because there was a TODO in the bottom of the screen saying the reason why we weren't using it. But because of the 'unused variable' warning, it wasn't checking for other bugs.
But I removed it (the eslint line), because I think it's better if it keeps complaining about it, so we don't forget it.